component/header.jsx

```diff
@@ -2,16 +2,17 @@ import { Link, Div, Color, Title } from './elements.jsx'
 import { roles } from '../data/discord.js'
 import { user } from '../lib/auth.js'
 import { HASH } from '../lib/env.js'
+import { useURL } from '../lib/router.js'
 
 const parseColor = (c) =>
   `rgb(${(c >> 16) & 0xff},${(c >> 8) & 0xff},${c & 0xff})`
 
-const LinkMatch = ({ match, children, ...props }) => (
+const LinkMatch = ({ match, children, path, ...props }) => (
   <li>
-    {match ? (
+    {path === props.href ? (
       <Color.Green>{children}</Color.Green>
     ) : (
-      <a {...props}>{children}</a>
+      <Link {...props}>{children}</Link>
```

Comment:
2 important things here:
- we use `Link` instead of `a` now, always
- You can easly check if the link is active if the current path match the href